runtime/lua/vim/filetype.lua

```diff
@@ -635,6 +639,12 @@ local extension = {
   tssop = "tssop",
   tutor = "tutor",
   twig = "twig",
+  ts = function(name, bufnr)
```

Comment:
It doesn't matter in this case because we're not using it, but let's call the first argument `path` (again, for future contributions that are just going to copy this implementation).

```suggestion
  ts = function(path, bufnr)
```